src/ast/Node.php

```diff
@@ -70,12 +70,7 @@ private function bindDecl($named_node, $type, $kind)
             throw new ScopeError(Localization::message('SCO130', [$named_node->name, $type]));
         }
 
-        // When it is a function and it is marked as public
-        if (Kind::K_FUNCTION === $kind && $named_node->is_pub) {
```

Comment:
You will add a keyword to replace the `pub` keyword, right? In this case, do you think that it is worth to delete it completely?